components/fxa-client/Cargo.toml

```diff
@@ -24,13 +23,17 @@ ffi-support = "0.4"
 viaduct = { path = "../viaduct" }
 rc_crypto = { path = "../support/rc_crypto", features = ["ece", "hawk"] }
 error-support = { path = "../support/error" }
+thiserror = "1.0"
+backtrace = "0.3"
 
 [dev-dependencies]
 viaduct-reqwest = { path = "../support/viaduct-reqwest" }
 cli-support = { path = "../support/cli" }
 dialoguer = "0.6"
 webbrowser = "0.5"
 mockiato = "0.9"
+anyhow = "1.0"
+
```

Comment:
nit: double new line before `[features]`, not sure if there's something that will format the `.toml` file but remove the extra line somehow